### ISSUE TITLE
Bars in chart not displayed when all values are the equal

### ISSUE THREAD

**Issue by SSebigo** [entronad/graphic#30]
Taking this code:

``` dart
List<int> getCandidates(int length) {
  final List<int> totalCandidatesList = <int>[];
  for (int i = 0; i < length; ++i) {
    totalCandidatesList.add(Random().nextInt(1000));
  }
  return totalCandidatesList;
}

List<Map<String, dynamic>> get distributionData {
  final List<int> tcd = getCandidates(21);

  final List<Map<String, dynamic>> chartData = <Map<String, dynamic>>[];
  for (int i = 0; i < tcd.length; ++i) {
    chartData.add({
      'index': i,
      'label': '$i',
      'value': 1,
    });
  }
  return chartData;
}
```

``` dart
Chart(
  data: distributionData,
  variables: {
    'label': Variable(
      accessor: (Map map) => map['label'] as String,
    ),
    'value': Variable(
      accessor: (Map map) => map['value'] as num,
    ),
    'index': Variable(
      accessor: (Map map) => map['index'] as num,
    ),
  },
  elements: [
    IntervalElement(),
  ],
  axes: [
    Defaults.horizontalAxis,
    Defaults.verticalAxis,
  ],
  selections: {'tap': PointSelection(dim: 1)},
  tooltip: TooltipGuide(
    backgroundColor: Palette.black,
    textStyle: TextStyles.caption.copyWith(color: Palette.white),
    variables: ['label', 'value'],
  ),
  crosshair: CrosshairGuide(),
)
```

Since all the values are == 1 (they just have to be the same values), the bars and the values on the Y axis won't show.

**Comment by entronad** [comment]
The default scale range is calculated by max difference of all data values, say the max value is 1 and min is 0, the range is calculated to [-0.1, 1.1], if the max value is 100 and min is 0, the range is [-10, 110]. It have to be like this.

So if all your data value are 1, which means the max difference is zero, the range is [1, 1], with zero interval.

In such case, you have to set the scale range manually:
```
'value': Variable(
  accessor: (Map map) => map['value'] as num,
  scale: LinearScale(min: 0, max: 2),
),
```

**Comment by entronad** [comment]
After thinking twice, we decide to add a range for the same data case, so that bars with same height will show. It will be in the next version.

Thanks for your advice!

**Comment by entronad** [comment]
The issue is fixed in the latest v0.6.2. Please have a try.

**Comment by SSebigo** [comment]
Seems to work, thx
